Engine/ScriptAnalyzer.types.ps1xml

```diff
@@ -50,7 +50,13 @@
       <ScriptProperty>
```

Comment:
Although it's possible to do this in the types file, I wonder if it's better to be done in the diagnostic record cs file